SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialPlayerSpawner.h

```diff
@@ -40,4 +39,6 @@ class SPATIALGDK_API USpatialPlayerSpawner : public UObject
 
 	FTimerManager* TimerManager;
 	int NumberOfAttempts;
+
+	TSet<FString> WorkersWithPlayersSpawned;
```

Comment:
I understand that this name is technical correct but can the set be called something that specifically refers to Clients that have already had players spawned?